packages/@okta/vuepress-site/docs/release-notes/2023/index.md

```diff
@@ -2,6 +2,64 @@
+
+#### OIN Manager support for Workflow Connector submission is GA in Production
+
+Okta [Workflows](https://www.okta.com/platform/workflows/) is a no-code, if-this-then-that logic builder that Okta orgs can use to automate custom or complex employee onboarding and offboarding flows in your application. You can now publish Workflow connectors that you create with the [Workflows Connector Builder](https://help.okta.com/okta_help.htm?type=wf&id=ext-connector-builder) in the Okta Integration Network (OIN) catalog. Publishing a Workflows connector with Okta allows your customers to deeply integrate your product with all other connectors in the catalog. Submit your Workflow connector by using the OIN Manager. See [Submit an integration](/docs/guides/submit-app/wfconnector/main/#submit-an-integration) for Workflows connectors. <!--OKTA-571943 & OKTA-573202-->
+
+#### OAuth 2.0 grant scopes added
+
+The OAuth 2.0 `okta.appGrants.manage` and `okta.appGrants.read` grant scopes are now available for use with the `/apps/{id}/grants` and `/apps/{id}/grants/{grantId}` endpoints.<!--OKTA-568970-->
+
+#### Scope parameter length increased
+
+The maximum length for the scope parameter of a refresh token request is now 4096 characters.<!--OKTA-573127-->
+
+#### OAuth 2.0 On-Behalf-Of Token Exchange is GA in Preview
+
+OAuth 2.0 On-Behalf-Of Token Exchange helps retain the user context in requests to downstream services. It provides a protocol approach to support scenarios where a client can exchange an access token received from an upstream client with a new token by interacting with the authorization server. See [Set up OAuth 2.0 On-Behalf-Of Token Exchange](/docs/guides/set-up-token-exchange/main/). <!--OKTA-572343 -->
+
+#### Configurable rate limits for OAuth 2.0 apps is GA in Production
+
+Rate limit violations mainly occur on authenticated endpoints. Currently, it isn't clear which OAuth 2.0 authenticated app consumes all the rate limits for an org. This increases the risk that one app consumes the entire rate limit bucket. To avoid this possibility, Okta admins can now configure how much rate limit capacity an individual OAuth 2.0 app can consume by editing the Application rate limits tab for each app. By setting a capacity on individual OAuth 2.0 apps, Okta admins have a new tool to monitor and investigate rate limit violations, and have the ability to view rate limit traffic generated by individual OAuth 2.0 apps. See [Rate limit dashboard bar graph](/docs/reference/rl-dashboard/#bar-graph). <!--OKTA-573387-->
+
+#### Bug fixed in 2023.04.0
+
+The Identity Sources API bulk upsert operation accepted an empty profile payload. (OKTA-533011)
```

Comment:
FYI ... I noticed that this bug was also tagged for OIE RNs, but doesn't appear. Is this an error on the ticket? Or should it be in OIE?

Reply:
Thanks @brianduffield-okta I followed the edited blurb spreadsheet. I see the OIE tag on the ticket. Will fix!

Reply:
Done.